docs/config.md

```diff
@@ -513,10 +513,11 @@ cannot be specified globally.
 
 Options available under this parameter are described in table below:
 
-| Option Name   | Description                                             | Supporting Types                                   |
-| :------------ | :------------------------------------------------------ | :------------------------------------------------- |
-| `concurrency` | Request concurrency used for generating the input load. | `<range>`, `<comma-delimited-list>`, or a `<list>` |
-| `batch_sizes` | Static batch size used for generating requests.         | `<range>`, `<comma-delimited-list>`, or a `<list>` |
+| Option Name    | Description                                             | Supporting Types                                   |
```

Comment:
Do we need to mention anywhere that only one of RR or concurrency can be specified?

Reply:
No, I don't think so. The user should already understand this, if they are going to be deliberately using request rate; and, if they don't they'll get a nice error message explaining that you can't use both.